src/lib/protocols/proof/messages/RequestPresentationMessage.ts

```diff
@@ -0,0 +1,51 @@
+import { Equals, IsArray, IsString, ValidateNested, IsOptional } from 'class-validator';
+import { AgentMessage } from '../../../agent/AgentMessage';
+import { MessageType } from './messages';
+import { Attachment } from '../../../decorators/attachment/Attachment';
+import { Expose, Type } from 'class-transformer';
+
+/**
+ * Interface for RequestPresentation
+ */
+export interface RequestPresentationData {
```

Comment:
We usually use `xxxOptions` naming convention for such interfaces. So it would be `RequestPresentationOptions`. Look at `messages` in `credentials`

Reply:
I also noticed, but in connection messages we use `xxxData`. My bad!

I already wanted to add this to the agenda for tomorrow so we can create a short list of standard practices. Hard to enforce standards if we don't have em 😅